src/connector/src/error.rs

```diff
@@ -42,6 +42,7 @@ def_anyhow_newtype! {
     url::ParseError => "failed to parse url",
     serde_json::Error => "failed to parse json",
     csv::Error => "failed to parse csv",
+    rust_decimal::Error => transparent,
```

Comment:
I think some context must be attached when converting a `rust_decimal::Error` to `ConnectorError`. If it's the case, could you remove the line here and explicitly call `.context()` on the result?

Reply:
fixed